### DISCUSSION TITLE
Error: Cannot read configuration

### DISCUSSION THREAD

**Discussion by erikschnaubelt** [iterate-ch/cyberduck#16347] [Q&A]
I am logged in to an S3 using Cyberduck 9.0.2 & Windows 11

After logging in I see all Files I am supposed to see. But anyntime I try to open a folder in the UI, an error message pops up saying:

```
Cannot read configuration
The <servername>.DNS is a network service that translates a server name to its Internet address. This error usually occurs if there is no internet connection or if there is a network configuration error. It is also possible that the DNS server is not responding or a firewall is preventing access to the network.
```

I spent enough time on the issue to safely say, internet connection, proxy or firewall are not the problem.. cyberduck version update did also not help.

If this error sounds familiar to you I am happy to receive some hint on what might cause the issue.

Cheers, Erik

**Comment by dkocher** [comment]
@erikschnaubelt Let us know what the resolution was to the issue you described.
